src/twisted/conch/client/default.py

```diff
@@ -130,6 +130,31 @@ def isInKnownHosts(host, pubKey, options):
 
 
 
+def getHostKeyAlgorithms(host, options):
```

Comment:
if that is the case, perhaps it makes sense to move it to a method on `KnownHostsFile`?  Perhaps good to do this before the next release?